crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -923,12 +923,12 @@ impl Room {
         self.inner.read().base_info.is_marked_unread
     }
 
-    /// Returns the recency timestamp of the room.
+    /// Returns the recency stamp of the room.
     ///
-    /// Please read `RoomInfo::recency_timestamp` to learn more.
+    /// Please read `RoomInfo::recency_stamp` to learn more.
```

Comment:
You should link to this if you'd like to reference docs on another struct. That being said, I don't think that you can sensibly link to the `recency_stamp` field since it's `pub(crate)` and not `pub` like this method.

```suggestion
    /// Please read [`RoomInfo::recency_stamp`] to learn more.
```

Reply:
Exactly, the field is not public so rustdoc would complain.

Reply:
Well yes, but how would the user find the docs as well, since you are referencing docs for a `pub(crate)` field?